docs/api/components/schemas.yaml

```diff
@@ -1308,7 +1308,7 @@ Database:
               description: |-
                 The maximum amount of memory the revision cache should take up in MB, setting to 0 will disable any eviction based on memory at rev cache.
                 When set this memory limit will work in in hand with revision cache size parameter. So you will potentially get eviction at revision cache both based off memory footprint and number of items in the cache.
-                **This is an enterprise-edition feature only**
+                **This is an Enterprise Edition feature only**
```

Comment:
There are a lot more instances of `enterprise-edition` than this one.
Would you like to correct them all in one go?

Reply:
Good point, shall do so!